etc/certs/generate.sh

```diff
@@ -37,6 +37,11 @@ openssl req -new -newkey rsa:2048 -nodes -keyout hostcert_xrd3.key.pem -subj "/C
 openssl x509 -req -days 9999 -CAcreateserial -extfile <(printf "subjectAltName=DNS:xrd3,DNS:localhost,DNS:xrd3.default.svc.cluster.local") -in hostcert_xrd3.csr -CA rucio_ca.pem -CAkey rucio_ca.key.pem -out hostcert_xrd3.pem -passin env:PASSPHRASE
 
 
+# XrootD Server 3
```

Comment:
4?

Reply:
Absolutely right. Fixed; also fixed in the DNS field a couple of lines lower.